### ISSUE TITLE
RFE: Inspect containers in RPMs

### ISSUE THREAD

**Issue by cipherboy** [rpminspect/rpminspect#30]
Similar to #29, I care about inspecting containers (tarballs, JARs, zips, &c) for new/deleted/modified files.

Again, for some this could be a lot of noise. My use case is JARs (a glorified ZIP archive) and telling what classes have been modified, added, or removed.

**Comment by dcantrell** [comment]
I recently added the javabytecode inspection that will check both .class files and .class files inside .jar containers.  The inspection looks to see that you match the expected JVM major version number for the release in question (JVM major version set in the configuration file and the release number is inferred from the disttag -or- explicitly set by the user on the command line).

This code could be moved to a separate function for additional jar examination.  I'm using libarchive, so we already have the ability to walk tarballs, zip files, and other container formats.  I would like to tie those specifically to different tests rather than just walking every embedded container.  JAR files are a good place to start.

**Comment by cipherboy** [comment]
Right, there are definitely security concerns with blindly walking every container you see. I'd suggest only walking a few formats and only top-level archives inside the RPM. I probably would mostly diff file lists -- the "modified" files is probably harder to do without extracting the entire archive.
